Documentation/Development.md

```diff
@@ -12,13 +12,13 @@ then run one of these commands:
 ##### macOS:
 
 ```sh
-$ swift/utils/build-script -R --llbuild --swiftpm
+$ ../swift/utils/build-script -R --llbuild --swiftpm
```

Comment:
Maybe we should also make it clear that we're inside the swiftpm directory. Something like: 
```
... and  then run one of these commands from the swiftpm directory ...:
```

Reply:
I added the information to README, what do you think?